builtin/methods.go

```diff
@@ -396,9 +396,7 @@ var MethodsPlaceholder = struct {
 var MethodsEthAccount = struct {
 	Constructor           abi.MethodNum
 	AuthenticateMessage   abi.MethodNum
-	UniversalReceiverHook abi.MethodNum
 }{
 	MethodConstructor,
 	MustGenerateFRCMethodNum("AuthenticateMessage"),
-	MustGenerateFRCMethodNum("Receive"),
```

Comment:
This is also needed for the Account actor in this file ~line 21

Reply:
added!